source/VisTrace.cpp

```diff
@@ -218,6 +218,48 @@ LUA_FUNCTION(RT_SetPixel)
 	return 0;
 }
 
+LUA_FUNCTION(RT_Load)
+{
+	LUA->CheckType(1, RenderTarget::id);
+	LUA->CheckType(2, Type::String);
+	// 3 and 4 are generateMips and scaleToRT respectively, they are both optional, but generateMips is true by default and cannot use the shorthand method for optional booleans.
+
+	const char* filepath = LUA->GetString(2);
+	bool generateMips = true;
+	if (LUA->IsType(3, Type::Bool)) {
+		generateMips = LUA->GetBool(3);
+	}
+
+	bool scaleToRT = LUA->GetBool(4);
+
+	RenderTarget* pRt = *LUA->GetUserType<RenderTarget*>(1, RenderTarget::id);
+	bool loaded = pRt->Load(filepath, generateMips, scaleToRT);
+
+	if (!loaded) {
+		LUA->ThrowError("Failed to load the image into the render target!");
+	}
+
+	return 0;
+}
+
+LUA_FUNCTION(RT_Save)
+{
+	LUA->CheckType(1, RenderTarget::id);
+	LUA->CheckType(2, Type::String);
+	
+	RenderTarget* pRt = *LUA->GetUserType<RenderTarget*>(1, RenderTarget::id);
```

Comment:
You should be checking if the RT is valid here
Despite checking IsValid in the method itself, you should check here to throw a more informative error message